src/ImageRunModal.jsx

```diff
@@ -355,6 +395,65 @@ export class ImageRunModal extends React.Component {
                 });
     }
 
+    onCreateClicked() {
+        const createConfig = this.getCreateConfig();
+        const { owner, runImage } = this.state;
+
+        // TODO: correct?
+        let isSystem = false;
```

Comment:
I think you can simplify this by pre-setting the state variable owner like this:
`this.state = { owner:  this.state.image.isSystem ? systemOwner : whaterverowner } `

Then here you can just do:
` const isSystem = owner === systemOwner`